sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/json/JsonSuite.scala

```diff
@@ -2435,6 +2435,24 @@ abstract class JsonSuite
     }
   }
 
+  test(s"SPARK-36477: infer json schema respect spark.sql.files.ignoreCorruptFiles") {
+    withSQLConf((SQLConf.IGNORE_CORRUPT_FILES.key, "true")) {
```

Comment:
I think we don't need this anymore. Let's also change testtitle, pr description etc.

Reply:
yes